### ISSUE TITLE
no links to chatrooms on lobby page

### ISSUE THREAD

**Issue by mseegan** [mseegan/attune#6]
I don't remember if we actually had these at one point, but there currently isn't any way to connect to a lobby other than having the url.

**Comment by LucklessXStar** [comment]
Yes, you are correct. We can make the index page the lobby if you want or have a splash page that introduces the website pointing to the lobby.

**Comment by mseegan** [comment]
you can still go to the pages, but theres no actual link.  for a first time
user it would be confusing.  but its really a quick fix.
